Networking/NetworkingTests/Mapper/OrderMapperTests.swift

```diff
@@ -370,6 +402,12 @@ private extension OrderMapperTests {
         return mapOrder(from: "order")
     }
 
+    /// Returns the OrderMapper output upon receiving `order-without-data`
```

Comment:
super nit:
```suggestion
    /// Returns the Order output upon receiving `order-without-data`
```

Reply:
Updated in 4d36b6c0b5bc68ed8e64894ee12394f7023b6d2b.